### PR TITLE
pass Json parameters and tags as variables

### PR DESCRIPTION
**Pull request by manishkumar3001**

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -10,6 +10,8 @@ resource "random_id" "id" {
 resource "cloudfoundry_service_instance" "redis" {
   name  = "tf-redis-${local.postfix}"
   space = var.cf_space_id
+  tags                           = var.tags
```
Comment (loafoe) [inline main.tf:13]: The tags does not seem to have a default making this param mandatory? Would be nice to have a sane default.

File: main.tf [hunk 2]
```diff
@@ -10,6 +10,8 @@ resource "random_id" "id" {
...
+  json_params                    = var.json_params
```
Comment (loafoe) [inline main.tf:14]: Does the default (empty string) work ok?

Comment (manishkumar3001) [reply main.tf]: I have given it a default value of empty string..

Comment (manishkumar3001) [reply main.tf]: I have also fixed the formating issue can you please queue this again..

File: main.tf (hunk 1 above)
Comment (manishkumar3001) [reply main.tf]: i have given default value for tags as empty list

File: main.tf (hunk 2 above)
Comment (manishkumar3001) [reply main.tf]: fixed the init issue.. it worked in my local 
Can you please queue again..??

**Pull request merged**